src/carrier_api/status.py

```diff
@@ -68,6 +68,8 @@ class Status:
     time_stamp: datetime = None
     zones: [StatusZone] = None
     raw_status_json: dict = None
+    raw_idu_status_json: dict = None # add for idu status
+    raw__odu_status_json: dict = None # add for odu status
```

Comment:
typo, two underlines in a row

Reply:
Thanks - changed and updated the pull request ( i think...i read/review a lot of jenkins github etc but rarely use)